chpl/chpl-service/src/main/java/gov/healthit/chpl/upload/certifiedProduct/CertifiedProductHandler2014.java

```diff
@@ -699,7 +699,7 @@ private void parseTestTools(PendingCertifiedProductEntity product, PendingCertif
 				ttEntity.setTestToolVersion(testToolVersion);
 				TestToolDTO testTool = testToolDao.getByName(testToolName);
 				if(testTool != null) {
-					if(testTool.isRetired() && product.getIcs().equals("false")) {
+					if(testTool.isRetired() && product.getIcs() == false) {

```

Comment:
The asBoolean() method gets called at CertifiedProductHandler2015.handle() line 78 in parseCertifiedProductDetails(). This is done before CertifiedProductHandler2015.parseCriteria().parseTestTools() at line 176. Thus, a check for asBoolean() in CertifiedProductHandler2014/2015.parseTestTools() would be redundant.